troubleshoot/ingest/opentelemetry/no-data-in-kibana.md

```diff
@@ -0,0 +1,108 @@
+---
+navigation_title: No data visible in Kibana
+description: Learn what to check when no data (logs, metrics, traces) appears in Kibana after setting up EDOT.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# No logs, metrics, or traces visible in Kibana
+
+If the EDOT Collector or SDKs appear to be running, but you see no logs, metrics, or traces in the {{kib}} UI, try to use these solutions to identify and resolve the issue.
+
+## Symptoms
+
+* Collector process is consuming CPU/memory but no telemetry data is visible in {{kib}}
+* APM services don’t show up in {{kib}}
+* Dashboards appear empty or partially loaded
+* The Collector is running without crash logs
+
+## Causes
+
+This issue is typically caused by one or more of the following:
+
+* Incorrect export endpoint
+* Missing or invalid API key/token
+* Network issues, such as proxy misconfigurations
+* TLS verification failures
+* Misconfigured pipelines or disabled signals
+* Incomplete setup — to capture all telemetry data, you must use the EDOT Collector together with an appropriate EDOT SDK
+
+## Resolution
+
+Use the following checks to identify and fix common configuration or connectivity issues that can prevent telemetry data from reaching {{kib}}.
+
+### Check export endpoint
+
+Make sure the Collector is configured to send data to the correct Elastic endpoint. For example, if using AWS, the endpoint should match this structure:
```

Comment:
shouldn't this be a check against the kibana shown endpoint, ie

1. open kibana for your cluster
2. go to "Add data" (top right in the Kibana window)
3. select "application" and click "OpenTelemetry"
4. in the "APM Agents" panel, select "OpenTelemetry" and check your endpoint is the same as the one shown